Formula/z/zuban.rb

```diff
@@ -21,9 +21,10 @@ class Zuban < Formula
   def install
     system "cargo", "install", *std_cargo_args(path: "crates/zuban")
 
-    (lib/"typeshed").mkpath
-    cp_r Formula["mypy"].opt_libexec.glob("lib/python*/site-packages/mypy/typeshed")[0].children, lib/"typeshed"
-    bin.env_script_all_files(libexec/"bin", ZUBAN_TYPESHED: lib/"typeshed")
+    typeshed = libexec/"lib/typeshed"
+    typeshed.mkpath
```

Comment:
Sorry one more thought, do you think we should work around `zubanls` not reading `ZUBAN_TYPESHED`? I believe that would just be


```suggestion
    # Workaround zubanls not reading ZUBAN_TYPESHED: https://github.com/zubanls/zuban/issues/53
    (typeshed = libexec/"lib/python3/site-packages/zuban/typeshed").mkpath
```

Reply:
Also, you can fix the failing CI by squashing your commits into one

Reply:
Due to the logic for finding typeshed relative to the executable, we have to keep it in `libexec`. Let's just keep `ZUBAN_TYPESHED` for now, pushed a fixup